addons/sale_project/views/project_update_template.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <template id="milestone_deadline_sale_project_inherit" inherit_id="project.milestone_deadline">
+        <xpath expr="//t[contains(@t-if, 'deadline')]" position="before">
+            <t t-if="milestone['quantity_percentage'] &gt; 0">
+                <t t-if="milestone['sale_line_name']">
```

Comment:
```suggestion
                <t t-if="milestone.sale_line_name">
```
Doesn't that work? If it does, please update all occurrences in this file :pray:

Reply:
Its dictionary not record and it rendered via python not js so it'll not work I guess :speak_no_evil:

Reply:
My bad again